package.json

```diff
@@ -27,7 +27,7 @@
     "build:es2018-cjs": "cross-env BABEL_ENV=cjs babel --config-file ./babel-es2018.config.js -x .mjs src -d es2018",
     "build:copyts": "copyfiles -f src/*.ts es5 && copyfiles -f src/*.ts es2015 && copyfiles -f src/*.ts es2018",
     "create:method": "node ./generate-yo method",
-    "verify": "run-s generate verify:* build test",
+    "verify": "run-s generate verify:* build test && npm run coverage",
```

Comment:
This is a [npm-run-all](https://www.npmjs.com/package/npm-run-all) command, it only takes patterns for the names of other scripts. This would just be `build test coverage`, except that coverage is running the tests, so just `build coverage`.

Reply:
Understood thanks for the explanation.

Reply:
is that `test:ts` or `test:js `?

Reply:
`test:ts`, because coverage is a form of `test:js`. Actually I think to make that abundantly clear the `coverage` script should be renamed to `test:js:coverage`